diff --git a/rotspeak/tap_rotspeak.c b/rotspeak/tap_rotspeak.c
index b637e71..a4b62b8 100644
--- a/rotspeak/tap_rotspeak.c
+++ b/rotspeak/tap_rotspeak.c
@@ -392,10 +392,10 @@ run_RotSpkr(LV2_Handle Instance,
                            cos_table[(unsigned long) phase_b_R]);
     }
 
-        ptr->phase_h += 1024.0f * freq_h * sample_index / ptr->sample_rate;
+    ptr->phase_h += 1024.0f * freq_h * sample_index / ptr->sample_rate;
     while (ptr->phase_h >= 1024.0f)
         ptr->phase_h -= 1024.0f;
-        ptr->phase_b += 1024.0f * freq_b * sample_index / ptr->sample_rate;
+    ptr->phase_b += 1024.0f * freq_b * sample_index / ptr->sample_rate;
     while (ptr->phase_b >= 1024.0f)
         ptr->phase_b -= 1024.0f;
 
@@ -548,10 +548,10 @@ run_adding_RotSpkr(LV2_Handle Instance,
                            cos_table[(unsigned long) phase_b_R]);
     }
 
-        ptr->phase_h += 1024.0f * freq_h * sample_index / ptr->sample_rate;
+    ptr->phase_h += 1024.0f * freq_h * sample_index / ptr->sample_rate;
     while (ptr->phase_h >= 1024.0f)
         ptr->phase_h -= 1024.0f;
-        ptr->phase_b += 1024.0f * freq_b * sample_index / ptr->sample_rate;
+    ptr->phase_b += 1024.0f * freq_b * sample_index / ptr->sample_rate;
     while (ptr->phase_b >= 1024.0f)
         ptr->phase_b -= 1024.0f;
 
diff --git a/tremolo/tap_tremolo.c b/tremolo/tap_tremolo.c
index 4258326..356f525 100644
--- a/tremolo/tap_tremolo.c
+++ b/tremolo/tap_tremolo.c
@@ -78,8 +78,8 @@ instantiate_Tremolo(const LV2_Descriptor * Descriptor, double SampleRate, const
 					((Tremolo *)ptr)->oldgain = 0.0f;
 	        if(flag == 0)
 	        {
-                for (i = 0; i < 1024; i++)
-                    cos_table[i] = cosf(i * M_PI / 512.0f);
+                    for (i = 0; i < 1024; i++)
+                        cos_table[i] = cosf(i * M_PI / 512.0f);
                     flag++;
 	        }
 		return ptr;
