diff --git a/src/Makefile.cardinal.mk b/src/Makefile.cardinal.mk
index f2caae90..34213d1d 100644
--- a/src/Makefile.cardinal.mk
+++ b/src/Makefile.cardinal.mk
@@ -94,7 +94,7 @@ endif
 
 RACK_EXTRA_LIBS  = ../../plugins/plugins.a
 RACK_EXTRA_LIBS += ../rack.a
-RACK_EXTRA_LIBS += $(DEP_LIB_PATH)/libquickjs.a
+#RACK_EXTRA_LIBS += $(DEP_LIB_PATH)/libquickjs.a
 
 ifneq ($(SYSDEPS),true)
 RACK_EXTRA_LIBS += $(DEP_LIB_PATH)/libjansson.a
