diff --git a/src/polyclouds.cpp b/src/polyclouds.cpp
index 54d9b80..772ea54 100755
--- a/src/polyclouds.cpp
+++ b/src/polyclouds.cpp
@@ -12,7 +12,7 @@
   OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 */
 
-#include "lv2/core/lv2.h"
+#include "lv2.h"
 
 /** Include standard C headers */
 #include <math.h>
diff --git a/src/polygrids.cpp b/src/polygrids.cpp
index 579187a..6a2f8eb 100755
--- a/src/polygrids.cpp
+++ b/src/polygrids.cpp
@@ -23,7 +23,7 @@
 #include "Metronome.hpp"
 #include "Oneshot.hpp"
 #include "TopographPatternGenerator.hpp"
-#include "lv2/core/lv2.h"
+#include "lv2.h"
 
 /** Include standard C headers */
 #include <math.h>
diff --git a/src/polymarbles.cpp b/src/polymarbles.cpp
index da6b5e9..85aa3ba 100644
--- a/src/polymarbles.cpp
+++ b/src/polymarbles.cpp
@@ -1,4 +1,4 @@
-#include "lv2/core/lv2.h"
+#include "lv2.h"
 
 /** Include standard C headers */
 #include <math.h>
diff --git a/src/polyplaits.cpp b/src/polyplaits.cpp
index 13a2245..c67fff9 100755
--- a/src/polyplaits.cpp
+++ b/src/polyplaits.cpp
@@ -13,7 +13,7 @@
 */
 
 #include "plaits/dsp/voice.h"
-#include "lv2/core/lv2.h"
+#include "lv2.h"
 
 /** Include standard C headers */
 #include <math.h>
diff --git a/src/polyrings.cpp b/src/polyrings.cpp
index 7bdab90..a73d67d 100644
--- a/src/polyrings.cpp
+++ b/src/polyrings.cpp
@@ -1,4 +1,4 @@
-#include "lv2/core/lv2.h"
+#include "lv2.h"
 
 /** Include standard C headers */
 #include <math.h>
diff --git a/src/polytides.cpp b/src/polytides.cpp
index c33685c..ac432f6 100644
--- a/src/polytides.cpp
+++ b/src/polytides.cpp
@@ -1,4 +1,4 @@
-#include "lv2/core/lv2.h"
+#include "lv2.h"
 
 /** Include standard C headers */
 #include <math.h>
diff --git a/src/polywarps.cpp b/src/polywarps.cpp
index 13440b5..eec2920 100755
--- a/src/polywarps.cpp
+++ b/src/polywarps.cpp
@@ -12,7 +12,7 @@
   OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 */
 
-#include "lv2/core/lv2.h"
+#include "lv2.h"
 
 /** Include standard C headers */
 #include <math.h>
