diff --git a/Makefile b/Makefile
index 78406ac..6d375d2 100755
--- a/Makefile
+++ b/Makefile
@@ -4,12 +4,7 @@
 # OPTIMIZATIONS ?= -mtune=cortex-a53 -funsafe-math-optimizations -ffast-math -fomit-frame-pointer -O3 -fno-finite-math-only -fvisibility=hidden -fdata-sections -ffunction-sections  -DNDEBUG # -fopt-info-vec-optimize
 PREFIX ?= /usr
 
-DEBUGFLAGS = -D_FORTIFY_SOURCE=2 -O0 -g -Wl,-z,relro,-z,now -fPIC -DPIC -Wall -D DEBUG -D NOSSE
-CFLAGS ?= $(OPTIMIZATIONS) $(DEBUGFLAGS)
-
 PKG_CONFIG?=pkg-config
-STRIP?=strip
-STRIPFLAGS?=-s
 
 polyclouds_VERSION?=$(shell git describe --tags HEAD 2>/dev/null | sed 's/-g.*$$//;s/^v//' || echo "LV2")
 ###############################################################################
@@ -24,6 +19,9 @@ BUNDLE=polylenticular.lv2
 BUILDDIR=build/
 targets=
 
+STRIP?=strip
+STRIPFLAGS?=-s
+
 UNAME=$(shell uname)
 ifeq ($(UNAME),Darwin)
   LV2LDFLAGS=-dynamiclib
@@ -147,25 +145,28 @@ $(BUILDDIR)$(LV2NAME_CLOUDS)$(LIB_EXT): $(CLOUDS_SOURCES)
 	$(CXX) $(CPPFLAGS) $(CFLAGS) $(FLAGS) \
 	  -o $(BUILDDIR)$(LV2NAME_CLOUDS)$(LIB_EXT) $(CLOUDS_SOURCES) \
 		-shared $(LV2LDFLAGS) $(LDFLAGS) $(LOADLIBES)
-	#$(STRIP) $(STRIPFLAGS) $(BUILDDIR)$(LV2NAME_CLOUDS)$(LIB_EXT)
+	$(STRIP) $(STRIPFLAGS) $(BUILDDIR)$(LV2NAME_CLOUDS)$(LIB_EXT)
 
 $(BUILDDIR)$(LV2NAME_WARPS)$(LIB_EXT): $(WARPS_SOURCES) 
 	@mkdir -p $(BUILDDIR)
 	$(CXX) $(CPPFLAGS) $(CFLAGS) $(FLAGS) \
 	  -o $(BUILDDIR)$(LV2NAME_WARPS)$(LIB_EXT) $(WARPS_SOURCES) \
 		-shared $(LV2LDFLAGS) $(LDFLAGS) $(LOADLIBES)
+	$(STRIP) $(STRIPFLAGS) $(BUILDDIR)$(LV2NAME_WARPS)$(LIB_EXT)
 
 $(BUILDDIR)$(LV2NAME_PLAITS)$(LIB_EXT): $(PLAITS_SOURCES) 
 	@mkdir -p $(BUILDDIR)
 	$(CXX) $(CPPFLAGS) $(CFLAGS) $(MUT_FLAGS) \
 	  -o $(BUILDDIR)$(LV2NAME_PLAITS)$(LIB_EXT) $(PLAITS_SOURCES) \
 		-shared $(LV2LDFLAGS) $(LDFLAGS) $(LOADLIBES)
+	$(STRIP) $(STRIPFLAGS) $(BUILDDIR)$(LV2NAME_PLAITS)$(LIB_EXT)
 
 $(BUILDDIR)$(LV2NAME_GRIDS)$(LIB_EXT): $(GRIDS_SOURCES) 
 	@mkdir -p $(BUILDDIR)
 	$(CXX) $(CPPFLAGS) $(CFLAGS) \
 	  -o $(BUILDDIR)$(LV2NAME_GRIDS)$(LIB_EXT) $(GRIDS_SOURCES) \
 		-shared $(LV2LDFLAGS) $(LDFLAGS) $(LOADLIBES)
+	$(STRIP) $(STRIPFLAGS) $(BUILDDIR)$(LV2NAME_GRIDS)$(LIB_EXT)
 
 # install/uninstall/clean target definitions
 
@@ -189,10 +190,6 @@ uninstall:
 	rm -f $(DESTDIR)$(LV2DIR)/$(BUNDLE)/$(LV2NAME_GRIDS)$(LIB_EXT)
 	-rmdir $(DESTDIR)$(LV2DIR)/$(BUNDLE)
 
-$(LV2NAME_CLOUDS)debug : clean $(RES_OBJECTS)
-	$(CXX) $(DEBUGFLAGS) $(OBJECTS) $(LDFLAGS) -o $(NAME).so
-	$(CC) $(DEBUGFLAGS) -Wl,-z,nodelete $(GUI_OBJECTS) $(RES_OBJECTS) $(GUI_LDFLAGS) -o $(NAME)_ui.so
-
 clean:
 	rm -f $(BUILDDIR)manifest.ttl $(BUILDDIR)$(LV2NAME_CLOUDS).ttl $(BUILDDIR)$(LV2NAME_CLOUDS)$(LIB_EXT) lv2syms
 	-test -d $(BUILDDIR) && rmdir $(BUILDDIR) || true
