From 011a83c50ec29f982476a42c1e7de1de18276a0d Mon Sep 17 00:00:00 2001
From: Massimo Pennazio <maxipenna@libero.it>
Date: Wed, 17 Aug 2022 00:41:42 +0200
Subject: [PATCH] Added support for Model SIMs

---
 mod/webserver.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/mod/webserver.py b/mod/webserver.py
index 1c1f1ad5..d62b309a 100644
--- a/mod/webserver.py
+++ b/mod/webserver.py
@@ -2111,6 +2111,9 @@ class FilesList(JsonRequestHandler):
         elif filetype == "sfz":
             return ("SFZ Instruments", (".sfz",))
 
+        elif filetype == "modelsim":
+            return ("Model SIMs", (".json",))
+
         else:
             return (None, ())
 
-- 
2.25.1

