From 7e1418481ff6a792e21bc2a8bf577223355bca67 Mon Sep 17 00:00:00 2001
From: Massimo Pennazio <maxipenna@libero.it>
Date: Tue, 11 Jan 2022 18:30:05 +0100
Subject: [PATCH 2/8] Migration to python3-pycryptodome

---
 mod/communication/crypto.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mod/communication/crypto.py b/mod/communication/crypto.py
index 2f077200..d91c90aa 100644
--- a/mod/communication/crypto.py
+++ b/mod/communication/crypto.py
@@ -20,7 +20,7 @@ def encrypt(recipient_key_txt: str, data: str):
 
     # Encrypt the data with the AES session keynonce: str, nonce: str,
     cipher_aes = AES.new(session_key, AES.MODE_EAX, uuid.uuid4().bytes)
-    ciphertext, tag = cipher_aes.encrypt_and_digest(data)
+    ciphertext, tag = cipher_aes.encrypt_and_digest(data.encode("utf8"))
     [out.write(x) for x in (cipher_aes.nonce, tag, ciphertext)]
     out.seek(0)
     return out.getvalue()
-- 
2.25.1

