diff --git a/mod-ams.lv2/env.ttl b/mod-ams.lv2/env.ttl
index e2b7852..ba022c0 100644
--- a/mod-ams.lv2/env.ttl
+++ b/mod-ams.lv2/env.ttl
@@ -31,7 +31,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 	guiext:ui <http://github.com/blablack/ams-lv2/env/gui> ;
 	lv2:optionalFeature lv2:hardRTCapable ;
 	lv2:port [
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 0 ;
 		lv2:symbol "gate" ;
 		lv2:name "Gate" ;
@@ -40,7 +40,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
         lv2:maximum 10.0 ;
         lv2:portProperty lv2:toggled ;
 	] ,	[
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 1 ;
 		lv2:symbol "retrigger" ;
 		lv2:name "Retrigger" ;
@@ -105,7 +105,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:minimum 1 ;
 		lv2:maximum 10 ;
 	] ,	[
-		a lv2:OutputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:OutputPort, lv2:CVPort ;
 		lv2:index 9 ;
 		lv2:symbol "out" ;
 		lv2:name "Out" ;
@@ -113,7 +113,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:minimum 0.0 ;
         lv2:maximum 10.0 ;
 	] ,	[
-		a lv2:OutputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:OutputPort, lv2:CVPort ;
 		lv2:index 10 ;
 		lv2:symbol "invOut" ;
 		lv2:name "Inverse Out" ;
diff --git a/mod-ams.lv2/lfo2_freq.ttl b/mod-ams.lv2/lfo2_freq.ttl
index c95ec49..9f7b19d 100644
--- a/mod-ams.lv2/lfo2_freq.ttl
+++ b/mod-ams.lv2/lfo2_freq.ttl
@@ -32,7 +32,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 	guiext:ui <http://github.com/blablack/ams-lv2/lfo2_freq/gui> ;
 	lv2:optionalFeature lv2:hardRTCapable ;
 	lv2:port [
-		a lv2:InputPort, lv2:CVPort, mod:CVPort;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 0 ;
 		lv2:symbol "reset" ;
 		lv2:name "Reset" ;
@@ -87,7 +87,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:minimum 0.0 ;
 		lv2:maximum 6.28 ;
 	] ,	[
-		a lv2:OutputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:OutputPort, lv2:CVPort ;
 		lv2:index 4 ;
 		lv2:symbol "output" ;
 		lv2:name "Output" ;
diff --git a/mod-ams.lv2/vcaexp.ttl b/mod-ams.lv2/vcaexp.ttl
index 7255bb4..2c990a1 100644
--- a/mod-ams.lv2/vcaexp.ttl
+++ b/mod-ams.lv2/vcaexp.ttl
@@ -30,7 +30,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 	guiext:ui <http://github.com/blablack/ams-lv2/vcaexp/gui> ;
 	lv2:optionalFeature lv2:hardRTCapable ;
 	lv2:port [
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 0 ; 
 		lv2:symbol "gain1Data" ;
 		lv2:name "Main Gain" ;
@@ -38,7 +38,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:minimum -5.0 ;
         lv2:maximum 5.0 ;
 	] ,	[
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 1 ; 
 		lv2:symbol "gain2Data" ;
 		lv2:name "2nd Gain" ;
diff --git a/mod-ams.lv2/vcalin.ttl b/mod-ams.lv2/vcalin.ttl
index e04d3ec..38ff175 100644
--- a/mod-ams.lv2/vcalin.ttl
+++ b/mod-ams.lv2/vcalin.ttl
@@ -31,7 +31,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 	guiext:ui <http://github.com/blablack/ams-lv2/vcalin/gui> ;
 	lv2:optionalFeature lv2:hardRTCapable ;
 	lv2:port [
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 0 ; 
 		lv2:symbol "gain1Data" ;
 		lv2:name "Main Gain" ;
@@ -39,7 +39,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:minimum -5.0 ;
         lv2:maximum 5.0 ;
 	] ,	[
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 1 ; 
 		lv2:symbol "gain2Data" ;
 		lv2:name "2nd Gain" ;
diff --git a/mod-ams.lv2/vcf.ttl b/mod-ams.lv2/vcf.ttl
index 6a16081..2f29331 100644
--- a/mod-ams.lv2/vcf.ttl
+++ b/mod-ams.lv2/vcf.ttl
@@ -37,7 +37,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:symbol "in" ;
 		lv2:name "In" ;
 	] ,	[
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 1 ; 
 		lv2:symbol "freqPort" ;
 		lv2:name "Frequency Mod" ;
@@ -45,7 +45,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:minimum -5.0 ;
 	        lv2:maximum 5.0 ;
 	] ,	[
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 2 ; 
 		lv2:symbol "expFM" ;
 		lv2:name "Exp FM" ;
@@ -53,7 +53,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:minimum -5.0 ;
 	        lv2:maximum 5.0 ;
 	] ,	[
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 3 ; 
 		lv2:symbol "linFM" ;
 		lv2:name "Lin FM" ;
@@ -61,7 +61,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:minimum -5.0 ;
 	        lv2:maximum 5.0 ;
 	] , 	[
-		a lv2:InputPort, lv2:CVPort, mod:CVPort ;
+		a lv2:InputPort, lv2:CVPort ;
 		lv2:index 4 ; 
 		lv2:symbol "resonancePort" ;
 		lv2:name "Resonance Mod" ;
diff --git a/mod-ams.lv2/vco3.ttl b/mod-ams.lv2/vco3.ttl
index ea6dadc..7fb9a26 100644
--- a/mod-ams.lv2/vco3.ttl
+++ b/mod-ams.lv2/vco3.ttl
@@ -53,7 +53,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 	guiext:ui <http://github.com/blablack/ams-lv2/vco3/gui> ;
 	lv2:optionalFeature lv2:hardRTCapable ;
 	lv2:port [
-		a lv2:CVPort, mod:CVPort, lv2:InputPort ;
+		a lv2:CVPort, lv2:InputPort ;
 		lv2:index 0 ; 
 		lv2:symbol "freq" ;
 		lv2:name "Frequency";
@@ -62,7 +62,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:maximum 5.0 ;
 		pg:group <http://github.com/blablack/ams-lv2/vco3#groupInput> ;
 	] ,	[
-		a lv2:CVPort, mod:CVPort, lv2:InputPort ;
+		a lv2:CVPort, lv2:InputPort ;
 		lv2:index 1 ; 
 		lv2:symbol "expFM" ;
 		lv2:name "Exp FM" ;
@@ -71,7 +71,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:maximum 5.0 ;
 		pg:group <http://github.com/blablack/ams-lv2/vco3#groupInput> ;
 	] ,	[
-		a lv2:CVPort, mod:CVPort, lv2:InputPort ;
+		a lv2:CVPort, lv2:InputPort ;
 		lv2:index 2 ; 
 		lv2:symbol "linFM" ;
 		lv2:name "Lin FM" ;
@@ -80,7 +80,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:maximum 5.0 ;
 		pg:group <http://github.com/blablack/ams-lv2/vco3#groupInput> ;
 	] ,	[
-		a lv2:CVPort, mod:CVPort, lv2:InputPort ;
+		a lv2:CVPort, lv2:InputPort ;
 		lv2:index 3 ; 
 		lv2:symbol "pwPort" ;
 		lv2:name "PW" ;
@@ -89,7 +89,7 @@ Ported from the AlsaModularSynth, a free and open-source project.
 		lv2:maximum 5.0 ;
 		pg:group <http://github.com/blablack/ams-lv2/vco3#groupInput> ;
 	] ,	[
-		a lv2:CVPort, mod:CVPort, lv2:InputPort ;
+		a lv2:CVPort, lv2:InputPort ;
 		lv2:index 4 ; 
 		lv2:symbol "sync" ;
 		lv2:name "Sync" ;
